### DISCUSSION TITLE
Paramétrage en https

### DISCUSSION THREAD

**Discussion by hansolo62** [dbarzin/mercator#201] [Q&A]
Bonjour
J essaye de passer l' application en https au travers du proxy apache et cela fonctionne mal.
Soit j ai des erreurs comme les handshake SSL ou alors un site à moitié https/http
Certains ont réussi à passer leur site en https et de quelle manière
Merci d' avance

**Comment by dbarzin** [comment]
Si vous utilisez Apache, vous devez activer RewriteEngine.
Voici un mien vers un exemple de configuration :
https://linuxize.com/post/redirect-http-to-https-in-apache/

**Reply by hansolo62** [reply]
Bonjour
merci beaucoup cela fonctionne à présent , j ai revu tous mes fichiers de conf d' apache
